dashboard/app/controllers/api/v1/assessments_controller.rb

```diff
@@ -58,8 +63,6 @@ def index
 
   # Return a hash by student_id for all students in a section. The value is a hash by
   # script_level_id, with information on the student responses for that assessment.
-  # Currently, very similar logic to section_assessments, but will change as we refactor. The format
-  # of the results is different enough that I'm duplicating some code it for now.
```

Comment:
This is no longer relevant?

Reply:
Yes, both because section_assessments seems to not be a thing anymore (I think it's referring to the index endpoint) and because they seem to be different now.